contrib/hadoop/src/main/java/org/xtreemfs/common/clients/hadoop/XtreemFSFileSystem.java

```diff
@@ -84,26 +84,14 @@ public void initialize(URI uri, Configuration conf) throws IOException {
             Logging.logMessage(Logging.LEVEL_DEBUG, this, "init : " + uri);
         }
 
-        String defaultURIString = conf.get("fs.defaultFS");
```

Comment:
As I remember correctly, we introduced fs.defaultFS as this seems to be the typical way to point to a default file system. Maybe @jan--f can clarify this. I would prefer to allow both options for compatibility purposes.